scripts/generate_version.py

```diff
@@ -0,0 +1,34 @@
+import argparse
+import os
+
+import versioneer
```

Comment:
So this won't be our vendored versioneer?

Reply:
Oops. I forgot to update this when I moved it into scripts, will fix it in my next PR.

Unfortunately, I don't think we are able to use non-vendored versioneer.